CHANGELOG.md

```diff
@@ -81,6 +81,7 @@ certificates. [[GH-13660](https://github.com/hashicorp/vault/pull/13660)]
 * identity: Fix possible nil pointer dereference. [[GH-13318](https://github.com/hashicorp/vault/pull/13318)]
 * identity: Fixes a panic in the OIDC key rotation due to a missing nil check. [[GH-13298](https://github.com/hashicorp/vault/pull/13298)]
 * kmip (enterprise): Fix locate by name operations fail to find key after a rekey operation.
+* physical/mysql: Create table with wider `vault_key` column when initializing database tables. [[GH-14231](https://github.com/hashicorp/vault/pull/14231)]
```

Comment:
`CHANGELOG.md` should actually not be updated directly. Changelog entries for a PR can be added by creating a changelog file for the associated PR. In your case, this would be `changelog/14231.txt`. The contents would then be:

````
```release-note:improvement
physical/mysql: Create table with wider `vault_key` column when initializing database tables.
```
````

Reply:
I've just push a fix for this, I hope everything looks good now.